.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,48 +3,35 @@ Resolves
 
 ## Submitter Checklist:
 
-- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
+- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
 - [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
-- [ ] Added/updated tests for this change (for new code or code which already has tests).
-- Verified that these changes build without errors on
-  - [ ] Android
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- Verified that these changes pass automated tests (unit, browser, security tests) on
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`, `npm run gn_check`)
+- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
+- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`
```

Comment:
One problem I commonly have is that I almost never have the browser tests working locally and it's never because of my changes. Maybe we should split that one into a separate item (or just leave it for CI)?

Reply:
same here! But I reckon it is still useful to run tests locally just in case you catch an error and save some CI cycles